Utils/github_workflow_scripts/close_stale_contrib_branches.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import urllib3
+import inspect
+from blessings import Terminal
+from github import Github, enable_console_debug_logging
+from github.Repository import Repository
+from typing import List
+from dateparser import parse
+from datetime import datetime, timezone
+from utils import timestamped_print
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+print = timestamped_print
+
+
+def get_stale_branch_names_with_contrib(repo: Repository) -> List[str]:  # noqa: E999
+    """Return the list of branches that have the prefix of "contrib/" without open pull requests
+    and that have not been updated for 2 months (stale)
+
+    Args:
+        repo (Repository): The repository whose branches will be searched and listed
+
+    Returns:
+        (List[str]): List of branch names that are stale and have the "contrib/" prefix
+    """
+    now = datetime.now(timezone.min)
+    organization = 'demisto'
+    branch_names = []
+    for branch in repo.get_branches():
+        # Make sure the branch is contrib
+        if branch.name.startswith('contrib/'):
+            prs_with_branch_as_base = repo.get_pulls(state='OPEN', base=branch.name)
+            prs_with_branch_as_head = repo.get_pulls(state='OPEN', head=f'{organization}:{branch.name})')
+
+            # Make sure there are no open prs pointing to/from the branch
+            if prs_with_branch_as_base.totalCount < 1 and prs_with_branch_as_head.totalCount < 1:
+                # load all members
+                inspect.getmembers(branch.commit)
+
+                # Make sure HEAD commit is stale
+                if (last_modified := branch.commit.last_modified) and (last_commit_datetime := parse(last_modified)):
+                    elapsed_days = (now - last_commit_datetime).days
+                    if elapsed_days >= 60:
+                        branch_names.append(branch.name)
+                else:
+                    print(f"Couldn't load HEAD for {branch.name}")
+    return branch_names
+
+
+def main():
+    debug_mode = len(sys.argv) >= 2 and 'debug' in sys.argv[1].casefold()
+    t = Terminal()
+    if debug_mode:
+        enable_console_debug_logging()
+    gh = Github(os.getenv('CONTENTBOT_GH_ADMIN_TOKEN'), verify=False)
+    organization = 'demisto'
+    repo = 'content'
+    content_repo = gh.get_repo(f'{organization}/{repo}')
+
+    stale_contrib_branches = list(set(get_stale_branch_names_with_contrib(content_repo)))
```

Comment:
Why are you doing `list(set(`? Are there duplicate branches returned? Doesn't seem to me like there should be.

Reply:
There shouldn't be duplicates, but initial runs yielded duplicates. I'm not sure why that would happen, might be an issue in the branches generator. In either case, I added this code just in case.

Reply:
Do you still see that happen? Anyway, it's fine to leave it, but then I don't think there is a need to cast it back to a list. Should be ok to leave it as a set.

Reply:
Let me run some more tests to make sure. I'll remove the list cast - the list was used mainly for the debug print, but it shouldn't matter, as both are readable.